src/Microsoft.PowerShell.Security/security/CredentialCommands.cs

```diff
@@ -86,14 +112,22 @@ public GetCredentialCommand() : base()
         /// </summary>
         protected override void BeginProcessing()
         {
-            if (!string.IsNullOrEmpty(this.Message))
+           string caption;
+            if(!string.IsNullOrEmpty(this.Title)) {
+                caption = this.Title;
+            }
+            else {
+                caption = UtilsStrings.PromptForCredential_DefaultCaption;
+            }
+            
+            if (!string.IsNullOrEmpty(this.Message) || !string.IsNullOrEmpty(this.UserName))
```

Comment:
if only `-Title` is specified, the code path seems cannot get to `this.Host.UI.PromptForCredential`